modules/aws-sqs/README.md

```diff
@@ -0,0 +1,170 @@
+# AWS SQS Module
+
+This module creates an SQS queue with optional API Gateway integration.
+
+## Usage
```

Comment:
This might be a good spot to document the additional changes that might be needed to get this to work with the airflow DAG. [Comment from that PR](https://github.com/Sage-Bionetworks-Workflows/orca-recipes/pull/87):

> I needed to give Airflow access to SQS in the dnt-dev AWS account so I created the airflow-sqs-access IAM user with the AmazonSQSFullAccess IAM role attached, and an AWS Secrets connection string airflow/connections/AWS_DNT_DEV_SQS_CONN in the dpe-prod account. The SQS infrastructure would need to be deployed in dpe-prod and similar access would need to be granted if we wanted to use this fully in production.

There is a bit of tight coupling that whomever uses this for the airflow dag would need to be aware of.